thcrap_proton

```diff
@@ -91,6 +92,17 @@ echo "---------------------------------------------------------"
 
 sh -c "$COMMAND" &
 
+if [ -n "$THPRAC_EXE" ]; then
+    # wait 5s and launch thprac so it could auto attach to the game
+    sleep 5
+    PROTON_CMD=$( echo "$COMMAND" | sed -n "s/^.*-- \(.*\) waitforexitandrun.*$/\\1/p" )
+    RUN_THPRAC="WINEPREFIX='$STEAM_COMPAT_DATA_PATH/pfx/' $PROTON_CMD run '$THPRAC_EXE'"
```

Comment:
Setting the ```WINEPREFIX``` is not necessary, since proton already uses the ```STEAM_COMPAT_DATA_PATH``` for this.

Reply:
Thanks, I didn't know this. Yes, removing `WINEPREFIX` still works.